src/weave/smoother.py

```diff
@@ -118,27 +118,29 @@ def dimensions(self, dimensions: List[Dimension]) -> None:
 
         self._dimensions = dimensions
 
-    def __call__(self, data: DataFrame, observed: Union[str, List[str]],
-                 smoothed: Optional[Union[str, List[str]]] = None,
+    def __call__(self, data: DataFrame, observed: str,
```

Comment:
I like `observed` is just one column now, simplify things a lot!

Reply:
Yes, it made things much easier for validation and the new standard deviation feature :)